packages/account-v2/src/containers/LoginHistoryTable/LoginHistoryTableCard.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { LoginHistory } from '@deriv/api-types';
+import { Text } from '@deriv-com/ui';
+import { formattedLoginHistoryData } from '../../utils';
+
+type TLoginHistoryProps = { loginHistory: LoginHistory };
+export const LoginHistoryTableCard = ({ loginHistory }: TLoginHistoryProps) => {
+    const formattedLoginHistory = formattedLoginHistoryData(loginHistory);
+    return (
+        <div className='inline-flex flex-col items-start gap-16'>
+            {' '}
+            {/* Loop through formatted login history data */}{' '}
+            {formattedLoginHistory.map((item, index) => (
+                <div
+                    className='shadow-md p-16 rounded-lg background: var(--system-light-8-primary-background, #FFF)'
+                    key={index}
```

Comment:
Should avoid using keys as index
https://blog.anja-stricker.de/why-using-index-as-key-in-reactjs-map-function-can-cause-problems-and-how-to-fix-them

Reply:
It's okay to have index as key if you are not doing any place swapping.

Reply:
In This case do i have to remove index or its alright ?

Reply:
![Screenshot 2024-04-16 at 2 26 33 PM (2)](https://github.com/binary-com/deriv-app/assets/158162395/b6d87584-ae88-442c-83e4-59439c690db4)


this is the error im facing while commiting and running the file